TombEngine/Objects/TR3/Entity/tr3_claw_mutant.cpp

```diff
@@ -309,10 +310,10 @@ namespace TEN::Entities::Creatures::TR3
 		if (item.HitPoints <= 0)
 		{
 			if (item.Animation.ActiveState != CLAW_MUTANT_STATE_DEATH)
-				SetAnimation(&item, CLAW_MUTANT_ANIM_DEATH);
+				SetAnimation(item, CLAW_MUTANT_ANIM_DEATH);
 
-			int frameEnd = GetAnimData(item, CLAW_MUTANT_ANIM_DEATH).frameEnd;
-			if (item.Animation.FrameNumber >= frameEnd)
+			int endFrameNumber = GetAnimData(item, CLAW_MUTANT_ANIM_DEATH).EndFrameNumber;
+			if (item.Animation.FrameNumber >= endFrameNumber)
```

Comment:
Can't we use TestLastFrame here?

Reply:
I agree. Let's not functionally change code in this PR, it should only substitute tests with full equivalent.